outbound.js

```diff
@@ -187,6 +211,18 @@
         return deferred.promise;
     };
 
+    Outbound.prototype.trackBatch = function(events) {
```

Comment:
Okay, what about if a `user` is passed instead of a `user_id`? As documented in [Updating a user with an event](http://docs.outbound.io/docs/track#updating-a-user-with-an-event)

It looks like the track method doesn't support that either, so I guess I'll ignore it for this pull request.